_dev/eslint-plugin-fxa/lib/configs/default.js

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
```

Comment:
interesting that this file uses semicolons but the client.js config above doesn't (although I know that's always differed between fxa-content-server and fxa-auth-server in the past) but I'm guessing the git hooks don't format or lint these files due to the globs in package.json: https://github.com/mozilla/fxa/blob/52933270b5640302dd20a6ae8bf0ad4ed6f63094/package.json#L59-L63